src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -518,45 +508,45 @@
         {
-          "description": "A string that contains a stable, opaque identifier for a rule.",
+          "description": "A stable, opaque identifier for the rule.",
           "type": "string"
         },
 
         "name": 
         {
-          "description": "An optional string that contains a rule identifier that is understandable to an end user.",
+          "description": "A rule identifier that is understandable to an end user.",
           "type": "string"
         },
 
         "shortDescription": 
         {
-          "description": "A string that contains a concise description of the rule. The short description property should be a single sentence that is understandable when displayed in user interface contexts where the available space is limited to a single line of text.",
+          "description": "A concise description of the rule. Should be a single sentence that is understandable when displayed in user interface contexts where the available space is limited to a single line of text.",
           "type": "string"
         },
 
         "fullDescription": 
         {
-          "description": "A string whose value is a string that describes the rule. The fullDescription property should, as far as possible, provide details sufficient to enable resolution of any problem indicated by the result.",
+          "description": "A string that describes the rule. Should, as far as possible, provide details sufficient to enable resolution of any problem indicated by the result.",
           "type": "string"
         },
 
         "options": 
         {
-          "description": "A dictionary consisting of a set of name/value pairs with arbitrary names. The options objects shall describe the set of configurable options supported by the rule. The value within each name/value pair shall be a string, which may be the empty string. The value shall not be a dictionary or sub-object.",
+          "description": "A dictionary consisting of a set of name/value pairs with arbitrary names. Describes the set of configurable options supported by the rule. The value within each name/value pair shall be a string, which may be the empty string. The value shall not be a dictionary or sub-object.",
           "type": "object"
         },
 
-        "formatSpecifiers": 
+        "messageFormats": 
         {
-          "description": "A dictionary consisting of a set of name/value pairs with arbitrary names. The value within each name/value pair shall be a string that can be passed to a string formatting function (e.g., the C language printf function) to construct a formatted message in combination with an arbitrary number of additional function arguments.",
+          "description": "A set of name/value pairs with arbitrary names. The value within each name/value pair shall consist of plain text interspersed with placeholders, which can be use to construc a formatted message in combination with an arbitrary number of additional string arguments.",
```

Comment:
Is the issue that you think I'm introducing an undefined term "formatted message"? I use that phrasing three times in the spec (although it doesn't appear in the glossary). *ponders* I'm ok with this as it stands. I don't think there can be any confusion about what it means.